cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -38,7 +38,7 @@ var addonsEnableCmd = &cobra.Command{
 }
 
 func init() {
-	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on when it is applied.")
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on. If multiple add-ons are enabled, this value is used by Minishift to determine the order in which to apply the add-ons.")
```

Comment:
Just "The priority of the add-on.". The second sentence is actually not accurate and also to long.